### PR TITLE
Revise descriptions, add tags, from Babel to Sphinx.

### PR DESCRIPTION
**Pull request by mvincerx**
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

**Comment by bryteise** [comment]
Rather than creating the five PRs, could you consolidate a multiple days of these kinds of changes into a single patch and PR? I'll get these merged as is though.

File: bundles/ImageMagick [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: ImageMagick
-# [DESCRIPTION]: ImageMagick
+# [DESCRIPTION]: Create, edit, or compose bitmap images.
```
Comment (bryteise) [inline bundles/ImageMagick]: I noticed that you sometimes have spaces at the end of lines, you don't need to fix this set of PRs but keep an eye out for that.

Comment (mvincerx) [reply bundles/ImageMagick]: OK.  Will do so.

**Comment by bryteise** [comment]
Merged.

**Pull request closed without merging**